gulpfile.js

```diff
@@ -41,42 +40,38 @@
 	// #region Script variables
 	let distPath = path.join(__dirname, "dist");
 	let srcPath = path.join(__dirname, "src");
-	let watchClose;
 
 	// If you specify environment variables to child_process, it overwrites all environment variables, including
 	// PATH. So, copy based on our existing env variables.
 	const env = process.env;
+	if (!env.PORT) {
```

Comment:
This should be after the check of NODE_ENV. and it should use the hash form. The what it was before the merge.